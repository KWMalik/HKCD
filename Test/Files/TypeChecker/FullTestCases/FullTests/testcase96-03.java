class Blub{
    public static void main(String[] args){}
}

class Ed{
    public int estimateNumberOfZombies(int limbs){
	return (limbs/4);
    }
}

class Ned extends Ed{}

class Fred extends Ned{}

class Bed extends Zed{}

class Jed extends Fred{}

class Cred extends Jed{}

class Zed extends Ced{}
