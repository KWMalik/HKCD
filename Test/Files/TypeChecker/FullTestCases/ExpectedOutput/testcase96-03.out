Superclass name Zed not in scope.
Superclass name Ced not in scope.
